functions.gs

```diff
@@ -285,3 +285,22 @@ function OCTOPART_GET_DESCRIPTION(mpn_or_sku, manuf) {
      return "(no description retrieved)";
   }
 }
+
+/**
+ * @param {string} mpn_or_sku is the search term. Search for components by manufacturer and/or part number. Part number terms may contain wildcards (“*”) but must also contain at least three non-wildcard characters.
+ * @param {string} manuf limits the search result to the specified manufacturer, if desired (default: no limitation)
+ * @returns {string}
+ * @customfunction
+ */
+function OCTOPART_GET_IMAGE(mpn_or_sku, manuf) {
```

Comment:
In addition, take a look at my comment on #3 regarding adding a parameter to get images of different sizes. Would be good to be able to choose what size you want directly from OCTOPART_GET_IMAGE.